website/content/en/docs/building-operators/ansible/reference/scaffolding.md

```diff
@@ -5,14 +5,131 @@ weight: 20
 ---
 
 After creating a new operator project using
-`operator-sdk new --type ansible`, the project directory has numerous generated folders and files. The following table describes a basic rundown of each generated file/directory.
+` operator-sdk init --plugins ansible.sdk.operatorframework.io/v1`,
+the project directory has many generated folders and files. The following
+table describes a basic rundown of each generated file/directory.
 
+| File/Folders   | Purpose |
+| :---           | :---    |
+| Dockerfile | The Dockerfile for building the container image for the operator. |
+| Makefile | Contains make targets for building, publishing, deploying the
+container image that wraps the operator binary, and make targets for
+installing and uninstalling the CRD. |
+| PROJECT | A YAML file containing meta information for the operator. |
+| config/crd | The base CRD file and the kustomization settings. |
+| config/default | A sidecar container to be injected to the controller
+manager for RBAC authorization against K8S API. | 
+| config/manager | The controller manager deployment. |
+| config/prometheus | The ServiceMonitor resource for monitoring the
+operator. |
+| config/rbac | The role, role binding for leader election and authentication
+proxy. |
+| config/sample | The sample resource created for the CRD. |
+| config/testing | Some sample configurations for testing. |
+| playbooks/ | A subdirectory for the playbook to run. |
+| roles/ | A subdirectory for the roles tree to run. |
+| watches.yaml | The Group, Version, and Kind of the resource to watch, and
+the Ansible invocation method. New entries are added via the 'create api'
+command. |
+| requirements.yml | A YAML file containing the Ansible collections and role
+dependencies to install during build. |
+| molecule/ | The [Molecule](https://molecule.readthedocs.io/) scenarios
+for end-to-end testing of your role and operator |
+
+
+## The Deployment
+
+The default Deployment manifest generated for the operator can be found in the
+`config/manager/manager.yaml` file. By default, the Deployment is named as
+'controller-manager' and it contains a single container named 'manager'. The
+Deployment will create a single Pod.
+
+For the container in the Pod, there are a few things to note. You will need
+to replace the image name and tag after you have built the Operator container 
```

Comment:
This is no longer necessary. The makefile uses kustomize to patch the image, so the users should be able to just run `make deploy IMG=<IMG>` and it will "just work"

Reply:
Got it.